app/views/candidate_interface/other_qualifications/type/new.html.erb

```diff
@@ -5,6 +5,12 @@
   <div class="govuk-grid-column-two-thirds">
     <%= form_with model: @qualification_type, url: candidate_interface_create_other_qualification_type_path do |f| %>
       <%= render partial: 'shared_form', locals: { f: f } %>
+
+      <% if @first_qualification %>
+        <p class="govuk-body"><%= govuk_link_to 'Return to your application', candidate_interface_application_form_path %></p>
+      <% else %>
+        <p class="govuk-body"><%= govuk_link_to 'Review qualifications', candidate_interface_review_other_qualifications_path %></p>
+      <% end %>
```

Comment:
I'm not 100% sure if this is the correct thing to do, but if a candidate deletes all of there qualifications they are sent to this page. It seems like there should be an easy way to return to their application form/review section if they do not want to add a qualification.